server/src/users/entities/user.entity.ts

```diff
@@ -21,7 +22,10 @@ export class User {
 
   @OneToMany(() => Asset, (asset) => asset.poster)
   assets: Asset[];
-  
+
+  @OneToMany(() => Transaction, (transaction)=> transaction.donater_user, { eager: true})
```

Comment:
remove eager

Reply:
also i'm seeing some missing whitespace 😲 
make sure you are up to date with master (`git pull origin main`, resolve conflicts, (if conflicts) `git add . && git merge --continue`)
and then run `npm run format` 
and might as well run `npm run checks` for good measure too

Reply:
git merge won't mess anything up on main?